modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -98,36 +98,36 @@
       end
     end
 
-    context 'backfill clinic service returns data' do
```

Comment:
FWIW, there are number of ways of disabling tests. For example, you can change put an x in front of the test block, like `xdescribe`, `xcontext`, or `xit`. Commenting them out is also fine for a temporary fix.

Reply:
👍